Source/MonoGame.Extended/Timers/ContinuousClock.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MonoGame.Extended.Timers
+{
+    public class ContinuousClock : IGameComponent
```

Comment:
I wouldn't really call it an advantage, just an alternative way of doing things within XNA/MonoGame.

From what I've read IGameComponent allows you to register the object as a component object with the Game class and as such will make the framework call the Update Method automatically on base.Update (http://stackoverflow.com/questions/15046181/what-is-base-updategametime-for-in-xna-games).

Some people prefer to do it that way, most though (I included) prefer to call Update Manually.